### ISSUE TITLE
Bug: Could not close lockfile in Windows 10 (with Python 3.9.7)

### ISSUE THREAD

**Issue by sdy623** [eoyilmaz/displaycal-py3#244]
I can't start this tool in Windows 10 with Python 3.9.7 properly. 
![image](https://user-images.githubusercontent.com/70808322/230707172-30df3a39-930a-4d28-82b9-53f540983ef2.png)
Here is the log.
```
Acquired lock file: <DisplayCAL.main.AppLock object at 0x0000017C61E62DF0>
start.py 0.0.0 1970-01-01T00:00:00Z
Windows 10 Enterprise LTSC 2021 Version 2009 Build 19044.2728 x86_64
Python 3.9.7 (default, Sep 16 2021, 16:59:28) [MSC v.1916 64 bit (AMD64)]
CA file c:\Users\Makoto\displaycal-py3\displaycal_venv\lib\site-packages\certifi\cacert.pem
Faulthandler
wxPython 4.2.0 msw (phoenix) wxWidgets 3.2.0
Encoding: utf-8
File system encoding: utf-8
Loading C:\Users\Makoto\AppData\Roaming\DisplayCAL\DisplayCAL.ini
listening
writing to lock file: port: 15411
Error - could not close lockfile C:\Users\Makoto\AppData\Roaming\DisplayCAL\DisplayCAL.lock: [Errno 13] Permission denied
```
The problem probably caused by this  `lock.flush()` in **line 464** `main.py`.

Any help for an easier installation in Windows would be appreciated.

**Comment by eoyilmaz** [comment]
The permission denied error is interesting, can you check the file permissions of C:\Users\Makoto\AppData\Roaming\DisplayCAL\DisplayCAL.lock

I've not run the tool in Windods yet. How did you install it?

**Comment by sdy623** [comment]
In fact, I can't find a way to install it in Windows, but I try to directly use python environment to run the DisplayCAL.pyw, `python start.py`, at first there is a bug 
```
Traceback (most recent call last):
  File "c:\Users\Makoto\displaycal-py3\start.py", line 2, in <module>
    from DisplayCAL.main import main
  File "c:\Users\Makoto\displaycal-py3\DisplayCAL\main.py", line 73, in <module>
    from util_win import win_ver
ModuleNotFoundError: No module named 'util_win'
```
I edited this line into `from .util_win import win_ver` in  `main.py` then the program got work temporarily, and get the error I reported

**Comment by sdy623** [comment]
Then I commeted this line `lock.flush()` try to fix it temporarily, and test it in Windows. It seems work, but I don't have the hardware to test its funtions.

**Comment by sdy623** [comment]
![image](https://user-images.githubusercontent.com/70808322/235310325-b168548d-7012-4594-aafc-047d056df145.png)

**Comment by AstralStorm** [comment]
The lock file implementation in DisplayCAL.util_os for Windows is highly suspect. It locks the whole file using a byte range lock, and when a file is locked, even the process owning the lock cannot write to the locked part.

**Comment by eoyilmaz** [comment]
This should be fixed with #312
